packages/build/src/log/logger.js

```diff
@@ -94,17 +94,23 @@ const logErrorSubHeader = function (logs, string, opts) {
   logSubHeader(logs, string, { color: THEME.errorSubHeader, ...opts })
 }
 
+// Print a sub-section header, when a warning happened
+const logWarningSubHeader = function (logs, string, opts) {
+  logSubHeader(logs, string, { color: THEME.warningSubHeader, ...opts })
+}
+
 module.exports = {
   getBufferLogs,
   log,
-  logError,
+  logWarning,
   logMessage,
   logObject,
   logArray,
-  logErrorArray,
```

Comment:
Is it not worth keeping `logErrorArray` in case we need to use it?

Reply:
In general, I would consider it dead code if not used.
We can always create it if we ever need it.
However, I am happy to keep it if you think that's better?

Reply:
I agree in principle. In this specific case, since we're talking about formatting utilities, I see this almost as defining a design system: we define what styles/formatting we make available, which may or may not be in use at any given point, just to avoid having to add and remove them whenever the use cases change.

Having said that, I don't have any strong preferences, so please do whatever you feel is best!

Reply:
:+1: 
I added it back in 8afc596b3754241790f8ca8d661420249fea147d